demos/index.js

```diff
@@ -5,10 +5,16 @@ const Powerpuff = types
     name: types.string,
     mood: types.enumeration(['happy', 'aggressive']),
   })
+  .actions(self => ({
+    setMood: (mood) => { self.mood = mood },
+  }))
 
 const powerpuff = Powerpuff.create({ name: 'Rebelle', mood: 'aggressive' })
 
 // /!\ doesn't work since we don't use an action to mutate the model
 // powerpuff.name = 'Belle'
 
+// does work because we use an action
+powerpuff.setMood('happy')
```

Comment:
This is bizarre to change the mood here and not into `demo / actions / 1` what do you think @delphinemillet ?

Reply:
I agree. What I propose to change in `demo / actions / 1` is : 
* the instance powerpuff becomes `rebelle` => `const rebelle = [...]`
* we set the mood to `happy` on instanciation => `Powerpuff.create({ name: 'Rebelle', mood: 'happy'})`
* we try to set the mood on the instance => `rebelle.mood = 'aggressive'`
And here we write `rebelle.setMood('aggressive')`
Seems more logical for the story telling 😃

Reply:
yes, I know, chnaging the instance name involves to update LOOOTS of issues 😅

Reply:
it is ok for me